HomeworkDeleagatesEvents/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HomeworkDeleagatesEvents
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            new CarWash("Bob's wash", 400);
+            new CarWash("Meri's wash", 200);
+
+            new WashingCard("Usual card", 1000, "31.12.20");
+            new WashingCard("Premium card", 2000, "31.12.25");
+            new WashingCard("Expired card", 500, "31.12.15");
```

Comment:
this kind of initialization isn't obvious at all 
Try to avoid this and use variables for saving class instances